lib/src/states/camera_picker_state.dart

```diff
@@ -199,12 +199,16 @@ class CameraPickerState extends State<CameraPicker>
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
     final CameraController? c = innerController;
+    if (c == null && state == AppLifecycleState.resumed) {
```

Comment:
Can we merge the null condition with the below one?

Reply:
This should work:
```dart
    if (c == null || !c.value.isInitialized) {
        if (state == AppLifecycleState.resumed) {
            initCameras(currentCamera);
        }
        else {
            // App state changed before we got the chance to initialize.
            return;
        }
    }
```

Let me try it now on my device.

Reply:
It worked and I updated the PR accordingly.